CONTRIBUTING.md

```diff
@@ -77,16 +77,8 @@ Before you submit your Pull Request (PR) consider the following guidelines:
     git push origin my-fix-branch
     ```
 
-* In GitHub, send a pull request to `typeorm:master`.
-* If we suggest changes then:
-  * Make the required updates.
-  * Re-run the TypeORM test suites to ensure tests are still passing.
-  * Rebase your branch and force push to your GitHub repository (this will update your Pull Request):
-
-    ```shell
-    git rebase master -i
-    git push -f
-    ```
+* In GitHub, send a pull request to `typeorm:master` and include a description following the template.
+* Changes can be added as new commits to the same branch, which will update your pull request.
```

Comment:
FYI, GitHub has this:
<img width="545" alt="image" src="https://github.com/user-attachments/assets/84cf4536-62dd-403f-a9d8-8fe56388e910" />